openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/AdvancedSearch.spec.ts

```diff
@@ -96,14 +96,8 @@ test.describe('Advanced Search', { tag: '@advanced-search' }, () => {
         tierTag2.responseData.fullyQualifiedName,
       ],
       'service.displayName.keyword': [table1.service.name, table2.service.name],
-      'database.displayName.keyword': [
-        table1.database.name,
-        table2.database.name,
-      ],
-      'databaseSchema.displayName.keyword': [
-        table1.schema.name,
-        table2.schema.name,
-      ],
+      'database.displayName': [table1.database.name, table2.database.name],
+      'databaseSchema.displayName': [table1.schema.name, table2.schema.name],
```

Comment:
Since we have now introduced the functionality for the display name, can we use database and schema display names and test that?

Reply:
will handle that in separate PR, currently this has test for changes made for Explore Tree.